pnpm-lock.yaml

```diff
@@ -17139,7 +17145,7 @@ packages:
     dev: true
 
   /strip-json-comments/2.0.1:
-    resolution: {integrity: sha1-PFMZQukIwml8DsNEhYwobHygpgo=, registry: https://registry.npmjs.com/, tarball: https://registry.npmjs.com/strip-json-comments/-/strip-json-comments-2.0.1.tgz}
```

Comment:
I'm not sure why this changed.

Reply:
I think this was changed when I isntall the eslint and @senecacdot/eslint-config-telescope

Reply:
I think this was changed when I isntall the eslint and @senecacdot/eslint-config-telescope